### PR TITLE
Conversions: JS error (#4551)

### PR DESCRIPTION
**Pull request by prattheev**
## Description of new Instant Answer, or changes

There was a misspelling in the conversions.js for decilitre unit.

## Related Issues and Discussions

Fixes [#4551](https://github.com/duckduckgo/zeroclickinfo-goodies/issues/4551)

## People to notify

@bsstoner @pjhampton 

---

Instant Answer Page: https://duck.co/ia/view/conversions

### CODE REVIEW

**Comment by daxtheduck** [comment]
Metadata placeholder post
            
*If you haven't already, please edit your first comment to include a link to the IA page at the bottom.*
            
---
            
**How to find your IA page**
- New Instant Answer
  -- follow the steps [here](https://duck.co/ia/new_ia) to create your IA page.
              
- Existing Instant Answer
  -- find your IA page from this [list](https://duck.co/ia)

File: share/goodie/conversions/conversions.js [hunk 1]
```diff
@@ -1,3 +1,39 @@
...
+ *  to put in a nuclear power plant, or anything else for that matter.
+ *  In fact, you'd be wise to print a copy, destroy all other copies,
+ *  including electronic copies, burn the printout, dissolve the ashes
+ *  in acid, and bury the waste into a hole in the desert.
+ *
+ *  --------------------------------------------------------------------------
+ */
+
+
+
```
Comment (pjhampton) [inline share/goodie/conversions/conversions.js]: Please remove all this.

File: share/goodie/conversions/conversions.js [hunk 2]
```diff
@@ -619,7 +655,7 @@ DDH.conversions = DDH.conversions || {};
                 { symbol: 'millilitre',     name: 'Millilitre' },
                 { symbol: 'hectolitre',     name: 'Hectolitre' },
                 { symbol: 'decalitre',      name: 'Decalitre' },
-                { symbol: 'deciliter',      name: 'Decilitre' },
+                { symbol: 'decilitre',      name: 'Decilitre' },
```
Comment (pjhampton) [inline share/goodie/conversions/conversions.js]: Have you tested this? I'm not sure this is the fix

File: share/goodie/conversions/conversions.js (hunk 1 above)
Comment (prattheev) [reply share/goodie/conversions/conversions.js]: Oops! Sorry. Forgot to disable my file-header. Will remove right away

File: share/goodie/conversions/conversions.js (hunk 2 above)
Comment (prattheev) [reply share/goodie/conversions/conversions.js]: Yup!

Test

I added a test case too incase.

**Comment by bsstoner** [comment]
thanks @prattheev

**Pull request merged**